### DISCUSSION TITLE
Quick review and few questions/suggestions

### DISCUSSION THREAD

**Discussion by Qronikarz** [spacedriveapp/spacedrive#2410] [General]
Hello.
I was checking the tag support in Spacedrive and stumbled upon few things.

Details: Linux Mint 21.3 Edge, Cinnamon edition; Spacedrive version: .deb v.0.2.13 (AppImage didn't launch)(Waiting for Flatpak)

1. It would be nice if Spacedrive used mouse hover effect in views other than media view.
2. Opening locations (location, devices, tags) from side panel sometimes requires clicking twice to show the files - first click it just switches the folder but is showing as empty and second click makes the files appear. Don't know what's happening.
3. When opening videos either in the quick view or the details view in right side panel the whole program crashes. (tested .mp4 and .webm)
4. I couldn't find the option to use AI labeling anywhere in the program. Is it exclusive to some OS?

And now the reason I was testing the Spacedrive - file tags.
Right now it reminds me of tags in macOS Finder. Which is not bad because it's one of the few default file browsers that supports the tags at all. In Spacedrive you can use custom tag names, make your own RGB color and view all tags where you can rename them and delete them.
![tags](https://github.com/spacedriveapp/spacedrive/assets/84787215/dd7edc61-9859-4d74-8f53-ad72e4fd9e8f)

1. When using the search bar there is the option to filter using tags. I tagged 3 files from my own program made for tagging media to see how the search works. You can see it above. All 3 were tagged with "lightbulb", 2 of them were tagged "on" and 2 of them were tagged "off". When I chose to filter "lightbulb" and "on" tags, it showed all 3 files which means that the tag search filter works in "boolean OR" mode. It would be extremely helpful to have an option to switch the boolean mode for searching.
2. Tags could be grouped to make managing them easier, for example you could make a group for scenery tags, group for item tags and all others.
3. Tags could keep track of how many files count.

If you want to take a look at other programs that offer file tagging and tags in general I have created a knowledge base as a repository in my account - TagsResearch. Naturally, Spacedrive is already in there and categorized with the template which I'll be updating when Spacedrive gains new features in future releases.

**Comment by utkubakir** [comment]
> I couldn't find the option to use AI labeling anywhere in the program. Is it exclusive to some OS?

we temporarily disabled AI labelling until some issues related to it are fixed

> When opening videos either in the quick view or the details view in right side panel the whole program crashes. (tested .mp4 and .webm)

does this apply for all video files or like specific encoding format?

**Comment by Qronikarz** [comment]
> does this apply for all video files or like specific encoding format?

I don't know. Just tested those 2 files few times and every time it crashed.
![a](https://github.com/spacedriveapp/spacedrive/assets/84787215/4cca1138-c7fb-4323-8a10-1144b8c1e16e)
![b](https://github.com/spacedriveapp/spacedrive/assets/84787215/27d1c004-1e3f-4359-937c-5beab0f64e82)

They seem to use different codecs? (screens from VLC Flatpak) Don't really know much about this area. I only know that they are web compatible since they were taken from here: https://github.com/Qronikarz/QuoMediaView/tree/main/example_qmv_base/files
